t/dedupe.c

```diff
@@ -473,9 +535,33 @@ static void show_chunk(struct chunk *c)
 	}
 }
 
-static void show_stat(uint64_t nextents, uint64_t nchunks, uint64_t ndupextents)
+static const char *capacity_unit[] = {"b","KB", "MB", "GB", "TB", "PB", "EB"};
+
+static void bytes_to_human_readable_unit(uint64_t n, uint64_t *n_out, const char **unit_out)
+{
+    uint8_t i;
+
+    *n_out = n;
+    i = 0;
+
+    if (*n_out < 1024) {
+        goto out;
+    }
+
+    while (*n_out >= 1024) {
+        i++;
+        *n_out /= 1024;
+    }
+
+out:
+    *unit_out = capacity_unit[i];
+}
+
+static void show_stat(uint64_t nextents, uint64_t nchunks, uint64_t ndupextents, uint64_t unique_capacity)
 {
 	double perc, ratio;
+    const char *unit;
+    uint64_t uc_human;
```

Comment:
Inconsistent indentation here.
```suggestion
	double perc, ratio;
	const char *unit;
	uint64_t uc_human;
```

Reply:
fixed